cmd/rook/main.go

```diff
@@ -155,7 +155,7 @@ func joinCluster() error {
 
 	services := []*clusterd.ClusterService{
 		ceph.NewCephService(cfg.devices, cfg.metadataDevice, cfg.directories,
-			cfg.forceFormat, cfg.location, clusterInfo.AdminSecret, cfg.storeConfig),
+			cfg.forceFormat, cfg.location, clusterInfo.AdminSecret, cfg.storeConfig, os.Getenv("HOSTNAME")),
```

Comment:
should this be a flag for standalone then with the default value being HOSTNAME env?